components/inputs/docs/input-numeric.md

```diff
@@ -148,11 +137,9 @@ numberInput.addEventListener('change', (e) => {
 * `inline-help`: Help text that will appear below the input. Use this only when other helpful cues are not sufficient, such as a carefully-worded label.
 <!-- docs: end hidden content -->
 
-### Accessibility Properties
-
-To make your usage of `d2l-input-percent` accessible, use the following properties when applicable:
-
-| Attribute | Description |
-|---|---|
-| `label` | **REQUIRED.** [Acts as a primary label on the input](https://www.w3.org/WAI/tutorials/forms/labels/). Visible unless `label-hidden` is also used. |
-| `label-hidden` | Use if label should be visually hidden but available for screen reader users. |
+## Accessibility
+- While `unit-label` is not mandatory by default, using the `unit` property makes it a required property
+- Despite being a lighter colour than the input text, the `unit` text still meets the [WCAG Minimum Contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) requirement of 4.5:1
+- It is important to note that `placeholder` is not a suitable replacement for `label` or any additional information, since it only applies when the input is empty, and not all users will be able to read it in the first place
```

Comment:
I'll be hiding most docs about placeholder today so you can remove this line.

Reply:
The first bullet made me want to know more about why it was setup that way, or how it affects the screen reader experience. Maybe we could add more context? Something like this:

> When using `'unit` make sure to also set `unit-label` since the `unit` itself is hidden from screen reader users; the `unit-label' is appended to the `label` off-screen so that it's announced by screen readers and hidden from sighted users.